src/monattack.cpp

```diff
@@ -2859,11 +2861,28 @@ bool mattack::stare( monster *z )
         if( g->u.sees( *z ) ) {
             add_msg( m_bad, _( "The %s stares at you, and you shudder." ), z->name() );
         } else {
-            add_msg( m_bad, _( "You feel like you're being watched, it makes you sick." ) );
+            add_msg( m_bad, _( "You feel like you're being watched; it makes you sick." ) );
```

Comment:
This is a very minor change, and yet it will break previous translation. Please revert.